pkg/cluster/profile_app.go

```diff
@@ -29,3 +30,26 @@ func CreateProfileApp(
 	}
 	return app, nil
 }
+func DestructProfileApp(
```

Comment:
Insert a blank line.

Reply:
Also, the function name should be DestroyProfileApps (Destroy instead of Destruct, and Apps instead of App because there could be more than one, at least in theory).